services/core-web/src/components/mine/Tailings/MineTailingsTable.js

```diff
@@ -237,4 +238,4 @@ const mapStateToProps = (state) => ({
   itrmExemptionStatusCodeHash: getITRBExemptionStatusCodeOptionsHash(state),
 });
 
-export default connect(mapStateToProps)(MineTailingsTable);
+export default withRouter(connect(mapStateToProps)(MineTailingsTable));
```

Comment:
Interesting.  Was `connect` causing problems here?

Reply:
@matbusby-fw Nope! `withRouter` adds `history` as a prop to the component which is used when navigating to the TSF details